Source/BugsnagSessionTracker.m

```diff
@@ -91,7 +91,7 @@ - (BugsnagSession *)runningSession {
 }
 
 - (void)startNewSessionIfAutoCaptureEnabled {
-    if (self.config.shouldAutoCaptureSessions  && [self.config shouldSendReports]) {
+    if (self.config.autoDetectSessions && [self.config shouldSendReports]) {
```

Comment:
I can't find the exact section in the spec but believe one of the points of discussion is that options like `shouldSendReports` should no longer affect sessions. This assumes that `shouldSendReports` has similar semantics to `autoDetectErrors` - is this the case? We may just want to remove the option entirely.